apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.js

```diff
@@ -208,7 +208,7 @@ GoogleBlockly.Extensions.registerMixin(
  * Derived from core Google Blockly:
  * https://github.com/google/blockly/blob/5a23c84e6ef9c0b2bbd503ad9f58fa86db1232a8/core/procedures.ts#L202-L287
  * @param {WorkspaceSvg} workspace The workspace containing procedures.
- * @returns an array of XML block elements
+ * @returns {import('blockly/core/utils/toolbox').FlyoutDefinition} an array of XML block elements
```

Comment:
Yep, we have to use this wonky syntax because `FlyoutDefinition` is only used in a comment, so the linter will think the import is unused. If we don't import anything and just write `{FlyoutDefinition}` here the type will be `any`. Therefore we import it directly so we can utilize the type in other files. Autocomplete will fill in this syntax thankfully so you don't have to memorize how to use it!

Reply:
Thank you for explaining!